chef/data_bags/crowbar/template-ipmi.json

```diff
@@ -3,19 +3,20 @@
   "description": "The default proposal for the ipmi barclamp",
   "attributes": {
     "ipmi": {
-	  "bmc_enable" : false,
-	  "bmc_user": "root",
-	  "bmc_password": "cr0wBar!",
+      "bmc_enable" : false,
+      "bmc_user": "root",
+      "bmc_password": "cr0wBar!",
+      "bmc_interface": "lan",
```

Comment:
The parameter should be used https://github.com/crowbar/crowbar-core/blob/master/crowbar_framework/app/models/node.rb#L982 here then. And "lanplus" as default would then be the way to go.